service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/AllergyStructureMapperTest.java

```diff
@@ -82,7 +92,7 @@ public void When_MappingAllergyIntoleranceJson_Expect_AllergyStructureXmlOutput(
 
         String outputMessage = allergyStructureMapper.mapAllergyIntoleranceToAllergyStructure(parsedAllergyIntolerance);
 
-        assertThat(outputMessage).isEqualTo(expectedOutputMessage);
+        assertThat(outputMessage).isEqualToIgnoringWhitespace(expectedOutputMessage);
```

Comment:
We can't use `isEqualToIgnoringWhitespace` in our mapper tests. There are places where it is signification e.g. notes / free text sections.